core/src/apps/misc/get_entropy.py

```diff
@@ -20,6 +20,23 @@ async def get_entropy(ctx: Context, msg: GetEntropy) -> Entropy:
         br_code=ButtonRequestType.ProtectCall,
     )
 
+    # WARNING: Only for testing purposes to spawn the passphrase dialogue
```

Comment:
you couldn't spawn the passphrase dialog because TR is not sending `PassphraseEntry` as a capability. if you change that in `base.py`, you'll get passphrase dialogs wherever appropriate (so like `trezorctl btc get-address`)

please modify `base.py` and remove the TEMP commit

Reply:
Done in [3ef66f8](https://github.com/trezor/trezor-firmware/pull/2277/commits/3ef66f8bb111a8672374191796c192f7c430def4) and [548460c](https://github.com/trezor/trezor-firmware/pull/2277/commits/548460c338506a70e0addb1861b263d965b72e15)